packages/@react-spectrum/s2/src/NotificationBadge.tsx

```diff
@@ -102,7 +106,7 @@ const badge = style({
     isIndicatorOnly: 'square',
     isSingleDigit: 'square'
   },
-  width: 'fit',
+  width: 'max',
```

Comment:
noticed that having width set to `fit` caused the text to wrap in locales like `'zh-CN-u-nu-hanidec'` so i changed it